app/manage/pods/device-details/share-modal-directive/index.js

```diff
@@ -11,10 +11,12 @@ module.exports = [
       controller: [
         '$scope',
         '$timeout',
-        function($scope, $timeout) {
+        '$location',
+        function($scope, $timeout, $location) {
+
           // template variables -------------------
           $scope.copySuccess = false;
-          $scope.url = window.location.hostname + window.location.pathname;
+          $scope.url = $location.absUrl().split('?')[0];
```

Comment:
questioning the necessity of split by query string, since we don't use that anywhere. It's not functionally crippling, just sayin' in case it wasn't intentional 
Will never happen in the wild, but I went to `http://localhost:5000/manage/?test/#/device/9544d25e-0565-4f27-9b66-870b0c02d471?demo=1` and then the link lead to `http://localhost:5000/manage/`- which isn't really a problem, I'm just ¯\_(ツ)_/¯